pkg/user/repository/DefaultAuthPolicyRepository.go

```diff
@@ -67,11 +50,11 @@ func (impl DefaultAuthPolicyRepositoryImpl) UpdatePolicyByRoleType(policy string
 	return &model, nil
 }
 
-func (impl DefaultAuthPolicyRepositoryImpl) GetPolicyByRoleType(roleType RoleType) (policy string, err error) {
+func (impl DefaultAuthPolicyRepositoryImpl) GetPolicyByRoleType(roleType bean.RoleType, accessType string) (policy string, err error) {
 	var model DefaultAuthPolicy
-	err = impl.dbConnection.Model(&model).Where("role_type = ?", roleType).Select()
+	err = impl.dbConnection.Model(&model).Where("role_type = ? AND access_type = ?", roleType, accessType).Select()
```

Comment:
lets not mix native query and go-pg methods. Use new Where() for different condition.